### DISCUSSION TITLE
vaultwarden --version tries to load config

### DISCUSSION THREAD

**Discussion by xoxys** [dani-garcia/vaultwarden#5045] [Q&A] [answered]
Starting with vaultwarden `v1.32.1`, `vaultwarden --version` attempts to load the configuration:

```
❯ docker run --entrypoint vaultwarden -e DATA_FOLDER=/fooo -it vaultwarden:devel --version
Error loading config:
  SQLite database directory `/fooo` does not exist or is not a directory
```

IMO `--version` should not try to load any configuration as it was with `v1.13.0`.

**Comment by BlackDex** [comment]
That is a known change. We also try to show the web-vault version. And for that we need the config. 

Any reason it shouldn't in your opinion?

**Reply by xoxys** [reply]
It's uncommon and might not exist. E.g. I build vaultwarden binaries in CI and run --help and --version to check the binaries. I can work around it if you don't plan to fix it.

**Reply by BlackDex** [reply]
It's not that uncommon, but ok.

But is see what you mean. It now aborts the version showing at all when there is zero config.

Could you create an issue for this please. Because that wasn't the way it was intended to work

**Reply by BlackDex** [reply, marked as answer]
Ah someone already did #5046

**Reply by xoxys** [reply]
Perfect, thanks :)
